### ISSUE TITLE
Transfer  from a worker to the main thread doesn't work in browser.

### ISSUE THREAD

**Issue by LeonidPolukhin** [josdejong/workerpool#454]
Worker returns Transfer object like
```
const array = new Float32Array(data);
return new workerpool.Transfer(array, [array.buffer]);
```

In worker.js: 
```
if (result instanceof Transfer) { 
    worker.send({
        id: request.id,
        result: result.message,
        error: null
    }, result.transfer);
}
```
but worker.send (message) takes only one argument skipping **transfer** so no transferring happens. See worker.js: 27
```
  worker.send = function (message) {
    postMessage(message);
  };
```

**Comment by KonghaYao** [comment]
@LeonidPolukhin 我同样遇到了这个问题，我将 workerpool 的版本改为 9.1.1 就可以正常运行了
"I encountered the same issue. I changed the version of workerpool to 9.1.1, and it runs normally now."

**Comment by josdejong** [comment]
@KonghaYao this should be fixed in v9.1.3 via PR #455. Can you elaborate?

**Comment by josdejong** [comment]
Closing due to a lack of response.
